extensions/sidekick/src/browser/history.ts

```diff
@@ -0,0 +1,132 @@
+import path from "path";
```

Comment:
Note that you can use `useSQL` hook from the `@raycast/utils` package to make SQL stuff easier to manage: https://developers.raycast.com/utilities/react-hooks/usesql 😊